src/megameklab/com/printing/RecordSheetTask.java

```diff
@@ -192,4 +187,41 @@ public Void doInBackground() throws Exception {
             return null;
         }
     }
+
+    /**
+     * Implementation of Pageable thar removes the record sheet objects as they are processed
```

Comment:
```suggestion
     * Implementation of Pageable that removes the record sheet objects as they are processed
```

Reply:
I guess I'm a few weeks late for Talk Like a Pirate Day.